### PR TITLE
feat: inside script training utils

### PR DESCRIPTION
**Pull request by sirtorry**
adds env var helpers.
partially resolves b/169782082

### CODE REVIEW

**Comment by sirtorry** [comment]
let me know if this implementation is satisfactory.

File: google/cloud/aiplatform/training_utils.py [hunk 1]
```diff
@@ -15,6 +15,57 @@
...
+
+    @property
+    def tensorboard_log_dir(self) -> str:
+        """
+        Returns: Cloud Storage URI of a directory intended for saving TensorBoard logs
+        """
+        return os.environ["AIP_TENSORBOARD_LOG_DIR"]
+
+    @property
+    def cluster_spec(self) -> str:
```
Comment (sasha-gitg) [inline google/cloud/aiplatform/training_utils.py:67]: This should return a `Optional[Dict]`.

After `os.environ.get` check for `None`. If not `None`, `json.loads` to deserialize the string.

Comment (sasha-gitg) [inline google/cloud/aiplatform/training_utils.py:67]: Additionally, add one more similar property for `TF_CONFIG`: https://cloud.google.com/ai-platform-unified/docs/training/distributed-training#tf-config

**Review by sasha-gitg — CHANGES_REQUESTED** [review]

**Comment by sirtorry** [comment]
current implementation returns None if env var not found. does that sound good?

**Review by dizcology — APPROVED** [review]

File: google/cloud/aiplatform/training_utils.py [hunk 2]
```diff
@@ -15,6 +15,91 @@
...
+    def tensorboard_log_dir(self) -> Optional[str]:
+        """
+        Returns:
+            Cloud Storage URI of a directory intended for saving TensorBoard logs.
+            None if environment variable not set.
+        """
+        return os.environ.get("AIP_TENSORBOARD_LOG_DIR")
+
+    @property
+    def cluster_spec(self) -> Optional[str]:
```
Comment (sasha-gitg) [inline google/cloud/aiplatform/training_utils.py:82]: Optional[Dict]

File: google/cloud/aiplatform/training_utils.py [hunk 3]
```diff
@@ -15,6 +15,91 @@
...
+            None if environment variable not set.
+        """
+        cluster_spec_env = os.environ.get("CLUSTER_SPEC")
+        if cluster_spec_env is not None:
+            return json.loads(cluster_spec_env)
+        else:
+            return None
+
+    @property
+    def tf_config(self) -> Optional[str]:
```
Comment (sasha-gitg) [inline google/cloud/aiplatform/training_utils.py:95]: Optional[Dict]

**Review by sasha-gitg — APPROVED** [review]
LGTM. Type hinting corrections.

**Pull request merged**